docs/integrations/spark.md

```diff
@@ -369,7 +374,10 @@ df.write.partitionBy("example-column").parquet(s"lakefs://${repo}/${branch}/outp
 
 The data is now created in lakeFS as new changes in your branch. You can now commit these changes or revert them.
 
-## Using the new OutputCommitter
+## Using the lakeFS OutputCommitter
+
+This feature is currently **in beta**.  It provides atomic writes and
+increases write performance.
```

Comment:
Not part of this review, but are we planning to document the hadoop 3 way of configuring the foc?

Reply:
Of course.  This is not a current priority.  Also I don't yet know how much I can make it become a default configuration.